models/genmodels.go

```diff
@@ -359,6 +359,14 @@ func (o %s) Error() string {
 `, errorType))
 	}
 
+	// default unknown response type is also an error type
+	buf.WriteString(`
+	func (u UnknownResponse) Error() string {
+		return fmt.Sprintf("unknown response with status: %%d body: %%s", u.StatusCode, u.Body)
```

Comment:
Was the code in samples generated with this? I'm not sure I understand the doubled-`%` since this isn't being `fmt.Printf`'d.

Reply:
Yes, this code generates what's in the samples directory. I don't quite know what's going on, but when I didn't have the double %s it showed up as `MISSING` and the double %s fixed it.